draft-dns-zone-digest.xml

```diff
@@ -614,17 +586,18 @@ example.com. 86400 IN &RRNAME; 2018031500 1 0 (
         </t>
         <t>
           In the placeholder record, the Serial field is
-          set to the current SOA Serial.  The Digest Type field is set
-          to the value for the chosen digest algorithm.
-          The Parameter field is set to a value whose meaning depends on the Digest Type.
+          set to the current SOA Serial.
+          The Scheme field is set to the value for the chosen collation scheme.
+          The Hash Algorithm field is set
+          to the value for the chosen hash algorithm.
           The Digest
           field is set to all zeroes and of length appropriate
-          for the chosen digest algorithm.
+          for the chosen hash algorithm.
```

Comment:
Erm. Do we need to note that out current solution only works for fixed length digest algorithms? Personally I don't think so, but mentioning in case.

Reply:
I don't think so either.  

But also I think variable-length algorithms could be accommodated in a couple of ways.  For example, the place holder for a variable length algorithm could be just use length of 1.

Reply:
True / WFM.